src/core/tests/type_prop/fft_base_complex.cpp

```diff
@@ -0,0 +1,514 @@
+        auto dft = std::make_shared<TypeParam>(data, axes);
+        FAIL() << "Node was created with invalid axes.";
+    } catch (const NodeValidationFailure& error) {
+        EXPECT_HAS_SUBSTRING(error.what(), "Each axis must be unique");
+    }
+
+    try {
+        auto axes = op::v0::Constant::create(element::i64, Shape{1}, {2});
+        auto dft = std::make_shared<TypeParam>(data, axes);
+        FAIL() << "Node was created with invalid axes.";
+    } catch (const NodeValidationFailure& error) {
+        EXPECT_HAS_SUBSTRING(error.what(), "Axis value: 2, must be in range (-3, 2)");
+    }
+
+    try {
+        auto axes = op::v0::Constant::create(element::i64, Shape{1, 2}, {0, 1});
+        auto dft = std::make_shared<TypeParam>(data, axes);
+        FAIL() << "Node was created with invalid axes.";
+    } catch (const NodeValidationFailure& error) {
+        EXPECT_HAS_SUBSTRING(error.what(), "Axes input must be 1D tensor.");
+    }
+}
+
+TYPED_TEST_P(FFTInvalidInput, invalid_signal_size) {
+    auto data = std::make_shared<op::v0::Parameter>(element::f32, Shape{4, 3, 2});
+    auto axes = op::v0::Constant::create(element::i64, Shape{1}, {0});
+
+    try {
+        auto signal_size = op::v0::Constant::create(element::i64, Shape{1, 2}, {0, 1});
+        auto dft = std::make_shared<TypeParam>(data, axes, signal_size);
+        FAIL() << "Node was created with invalid signal size.";
+    } catch (const NodeValidationFailure& error) {
+        EXPECT_HAS_SUBSTRING(error.what(), "Signal size input must be 1D tensor.");
+    }
+
+    try {
+        auto signal_size = op::v0::Constant::create(element::i64, Shape{2}, {0, 1});
+        auto dft = std::make_shared<TypeParam>(data, axes, signal_size);
+        FAIL() << "Node was created with invalid signal size.";
+    } catch (const NodeValidationFailure& error) {
+        EXPECT_HAS_SUBSTRING(error.what(), "Sizes of inputs 'axes' and 'signal_size' must be equal.");
+    }
+}
+
+REGISTER_TYPED_TEST_SUITE_P(FFTInvalidInput, invalid_input_data, invalid_axes, invalid_signal_size);
+INSTANTIATE_TYPED_TEST_SUITE_P(type_prop, FFTInvalidInput, FFTBaseTypes);
+
+template <class TOp>
+class FFTDynamicTypes : public TypePropOpTest<TOp> {};
+
+TYPED_TEST_SUITE_P(FFTDynamicTypes);
+
+TYPED_TEST_P(FFTDynamicTypes, dynamic_types) {
+    const auto input_shape = PartialShape{2, 180, 180, 2};
+    const auto axes_shape = PartialShape::dynamic();
+    const auto signal_size_shape = PartialShape::dynamic();
+    const auto ref_output_shape = PartialShape{Dimension::dynamic(), Dimension::dynamic(), Dimension::dynamic(), 2};
+
+    auto data = std::make_shared<op::v0::Parameter>(element::dynamic, input_shape);
+    auto axes_input = std::make_shared<op::v0::Parameter>(element::dynamic, axes_shape);
+    auto signal_size_input = std::make_shared<op::v0::Parameter>(element::dynamic, signal_size_shape);
+    auto dft = std::make_shared<TypeParam>(data, axes_input, signal_size_input);
```

Comment:
```suggestion
    auto dft = this->make_op(data, axes_input, signal_size_input);
```
?
Check similar cases in modified tests and/or test fixtures.

Reply:
Usage of <TypeParam> is not wrong, but okay, updated;)